package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -507,7 +506,11 @@ class HydrogenBondLifetimes(object):
        may have failed in some cases.
     """
 
-    def __init__(self, universe, selection1, selection2, t0, tf, dtmax):
+
+    def __init__(self, universe, selection1, selection2, t0, tf, dtmax,
+                 nproc=1):
+        #warnings.warn("This class is deprecated; use analysis.hydrogen_bonds.hbond_analysis.HydrogenBondAnalysis ",
```

Comment:
Is the class not deprecated anymore? If so, remove the comments.

Reply:
We aim to depricate it after this PR is merged, due to several problems with the current implementation. Removed the comment for now.